nilearn/glm/first_level/first_level.py

```diff
@@ -1811,6 +1811,10 @@ def _get_events_files(
     events : :obj:`list` of :obj:`str`
         List of fullpath to the events files
     """
+    # pop the 'desc' filter
+    # it would otherwise trigger some meaningless warnings
+    # as desc entity are not supported in BIDS raw datasets
+    img_filters = [x for x in img_filters if x[0] != "desc"]
```

Comment:
these should reduce the number of warnings we get when running the tests but also reduce warnings users get when filtering BIDS dataset with the desc entity